20_2 Scheduler scrollTo: create description (#1687)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxScheduler/1 Configuration/scrolling.md

Configures scrolling.

---
Scrolling allows a user to browse data left outside the current viewport. The UI component contains two scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.

> - [scrollTo(date, group, allDay)](/Documentation/ApiReference/UI_Widgets/dxScheduler/Methods/#scrollTodate_group_allDay)

Reviewer: ```suggestion
#####See Also#####
- [scrollTo(date, group, allDay)](/Documentation/ApiReference/UI_Widgets/dxScheduler/Methods/#scrollTodate_group_allDay)
```

Configures scrolling.

---

> Scrolling allows a user to browse data left outside the current viewport. The UI component contains two scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.

Reviewer 2: ```suggestion
Scrolling allows a user to browse data outside the current viewport. Information about the two scrolling modes is available in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.
```
